extension.js

```diff
@@ -112,25 +118,6 @@ function updateSnippetFile(snippetFilename, jsonData) {
             if (err) {
                 throw err;
             }
-
-            var snippetComment = fs.readFileSync(SNIPPET_TEMPLATE_NAME, 'utf8').replace('$(languageId)', path.basename(snippetFilename));
-            var current = fs.readFileSync(snippetFilename, 'utf8');
-            if (!current.startsWith(snippetComment)) {
-                prependFile.sync(snippetFilename, snippetComment, {encoding: 'utf8'});
-            }
-            vscode.workspace.openTextDocument(snippetFilename).then(function(doc) {
-                vscode.window.showTextDocument(doc, vscode.ViewColumn.Two)
-                .then(selectDefaultSnippet);
-            });
```

Comment:
I'm not a fan of opening the snippet pane, but maybe that's just me. I didn't know what had happened at first and it felt like my workspace was messed with.